### DISCUSSION TITLE
Better view of threads and Tweet&Replies section

### DISCUSSION THREAD

**Discussion by Edodaddo** [robertodoering/harpy#700] [Ideas]
Two feature requests that are very important to me (Pro user btw):

- Better view of threads;  very often they are impossible to read on Harpy and I am forced to open Twitter to read them (eg. https://twitter.com/IAmClintMurphy/status/1539961255737233408?t=99WKvoCD8Bf7a93rZ4v7pg&s=19)

- Addition of "Tweets & Replies" section in Twitter profiles;  every time I tap on an interesting profile it is a real mess to understand  if what I'm watching is a tweet from him/her or just a reply to a tweet

Thank you for your work, I hope that in the future these features can become reality!  :)

**Comment by simanchal-45** [comment]
Completely agree.

**Comment by monee121** [comment]
Absolutely. Sometimes most of the thread will show, then all of a sudden it will say "no replies exist" when I know for sure there are more. I do a lot of 30 question threads and I have to use Twitter app for them to get all the questions.

**Reply by yoshimo** [reply]
#322  probably helps here a lot

**Comment by scottiefalkon** [comment]
It should. On the current API there's nothing that can be done.
